### PR TITLE
(Re-open) Update controller_configuration.rst.

### PR DESCRIPTION
**Pull request by 130s**
Review comments in #56 applied.

### CODE REVIEW

File: doc/pr2_tutorials/planning/src/doc/controller_configuration.rst [hunk 1]
```diff
@@ -96,3 +96,14 @@ The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot mus
...
 
 You should also be able to see (using *rostopic info topic_name*) that the topics are published/subscribed to by the controllers on your robot and also by the *move_group* node.
+
+Remapping /joint_states topic
+-----------------------------
+
+When you have a program that uses move group, you may need to remap the topic /joint_states to /robot/joint_states, otherwise MoveIt! won't have feedback from the joints. To do this remapping you could make a simple launch file for your node as follows ::
+
+<launch>
+<node name="any_name" pkg="your_package_name" type="your_executable_name" output="screen" args="joint_states:=/robot/joint_states"/>
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/doc/controller_configuration.rst]: Please replace this by
```
<node name="any_name" pkg="your_package_name" type="your_executable_name" output="screen">
   <rename from="joint_states" to="robot/joint_states"/>
</node>
```

File: doc/pr2_tutorials/planning/src/doc/controller_configuration.rst [hunk 2]
```diff
@@ -96,3 +96,14 @@ The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot mus
  * /gripper_controller/gripper_action/result
 
 You should also be able to see (using *rostopic info topic_name*) that the topics are published/subscribed to by the controllers on your robot and also by the *move_group* node.
+
+Remapping /joint_states topic
+-----------------------------
+
+When you have a program that uses move group, you may need to remap the topic /joint_states to /robot/joint_states, otherwise MoveIt! won't have feedback from the joints. To do this remapping you could make a simple launch file for your node as follows ::
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/doc/controller_configuration.rst]: "that uses move group" is pretty fuzzy here, but I never encountered the problem so I wouldn't know how to improve the wording.

File: doc/pr2_tutorials/planning/src/doc/controller_configuration.rst (hunk 1 above)
Comment (130s) [inline doc/pr2_tutorials/planning/src/doc/controller_configuration.rst]: Done. 
I actually specified the node type as `move_group`, hoping that's what we need in this passage.

File: doc/pr2_tutorials/planning/src/doc/controller_configuration.rst (hunk 2 above)
Comment (130s) [inline doc/pr2_tutorials/planning/src/doc/controller_configuration.rst]: I wasn't sure how to address your comment but tried. Please see the updated commit.

**Comment by 130s** [comment]
Ok, merging then.
@ghanimmukhtar if you have any comments to https://github.com/ros-planning/moveit_tutorials/pull/66#issuecomment-283054279 we'll appreciate if you share with us.

**Comment by 130s** [comment]
Cherry-picked to indigo-devel https://github.com/ros-planning/moveit_tutorials/commit/d0b91ab89d696d409a7358b5dcfed60b01c6b6c5

**Comment by ghanimmukhtar** [comment]
@130s I understood that you want me to comment on @v4hn comment, so here it goes. I think he is absolutely right that this should affect all nodes that use a `CurrentStateMonitor`, and to be more precise, I believe the genesis of this issue comes from the variable `const std::string planning_scene_monitor::PlanningSceneMonitor::DEFAULT_JOINT_STATES_TOPIC = "joint_states";`
which is defined, as mentioned in his comment, in the class `PlanningSceneMonitor` and used in the upstream nodes. Moreover, for the Baxter robot (and may be for other robots), and for some reason, the topic name is `/robot/joint_states` hence the need for remapping. 

Cheers!

**Comment by davetcoleman** [comment]
+1 thanks @130s !

**Pull request merged**